src/core/client.ts

```diff
@@ -24,11 +24,11 @@ const notifier = {
   version: '__VERSION__'
 }
 
-// Split at commas
-const TAG_SEPARATOR = /,/
+// Split at commas and spaces
+const TAG_SEPARATOR = /,|\s/
 
-// Removes any non-word characters
-const TAG_SANITIZER = /[^\w]/g
+// Removes extra whitespace from individual tags
+const TAG_SANITIZER = /\s/g
```

Comment:
I don't think this is necessary anymore. If I'm thinking this correctly, by running the `TAG_SEPARATOR` first, there wouldn't be any individual tags with whitespace.

Reply:
Update: To confirm my theory, I just tested removing usages of `TAG_SANITIZER` on this branch and all the tests pass.